src/resolve/resolved-dns-transaction.h

```diff
@@ -1,8 +1,10 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-#include "sd-event.h"
 #include "in-addr-util.h"
+#include "sd-event.h"
+#include <curl/curl.h>
```

Comment:
please place this first.

the rule of ordering includes is:

1. first foreign public headers i.e. `#include <…>` lines
2. then our own public headers, i.e. `#include "sd-*.h` lines
3. finally our own internal heaers, i.e. all others matching `#include "*.h"`